### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by AngusJohnson** [graphics32/graphics32#221]
The current documentation isn't that easy to read (or navigate) on desktop displays, but it's almost impossible to read on mobile devices without constant zooming  and scrolling.

While I'm no longer an active contributor to Graphics32, I've recently done a major update to DocProcessor which IMHO addresses many of its deficiencies. I've also made relatively minor changes to the current Graphics32 help source files (link below) so it compiles with the updated DocProcessor.

Here's the link to the current [online documentation](https://graphics32.github.io/Docs/Overview/_Body.htm).
Here's the [very temporary link](http://www.angusj.com/Gr32/Docs/Overview.htm) to the same documentation which has been compiled using my updated DocProcessor.

And here are 2 very scaled down screenshots from my mobile phone that show an example of the current and 'new' documentation, where you can see how the menu, and image and table all scale appropriately on mobile devices in the newly compiled documentation:
![screenshot_old](https://user-images.githubusercontent.com/5280692/221462479-ee8a6bfb-2549-4650-bfa9-c4ca6ef7583d.jpg) ![Screenshot_20230303-203037_Chrome](https://user-images.githubusercontent.com/5280692/222697923-0b79a734-3558-483e-8dcd-9634a656dd2d.jpg)

And here are the updated files:
[DocProcessor Binary and Source](https://github.com/graphics32/graphics32/files/10835740/DocProcessor_2.0.1.zip)
[Gr32 Documentation Files](https://github.com/graphics32/graphics32/files/10835744/Gr32_Documentation.zip)

![DocProcessor](https://user-images.githubusercontent.com/5280692/221386009-4297ecc8-b987-46bc-86f8-ebafa28aaf40.png)
Note: In this updated DocProcessor, I've removed the code that compiles CHM files. Does anyone still use these?

**Comment by AngusJohnson** [comment]
Now you just need to update the [**online help docs**](https://graphics32.github.io/Docs/Overview/_Body.htm) 😁.

[Gr32_Documentation_New.zip](https://github.com/graphics32/graphics32/files/14320390/Gr32_Documentation_New.zip)
[Gr32_New_Docs_Only.zip](https://github.com/graphics32/graphics32/files/14320391/Gr32_New_Docs_Only.zip)

**Comment by andersmelander** [comment]
Thanks - Yeah, I guess I should get that done too.

I primarily wanted to get the source updated so your work wasn't lost. I will upload your updated documentation too but I'm not going to maintain it using DocProcessor going forward as I simply refuse to use notepad (or whatever) to main the documentation. Life's too short for that. I need WYSIWYG.

As I've [mentioned elsewhere](https://github.com/graphics32/graphics32/issues/76#issuecomment-1176154202) I have a version of the updated documentation made in Wordpress but I have so far been unable to [generate a static site](https://wordpress.org/plugins/simply-static/) from it (i.e. a set of HTML files that I can upload). I have the class library documentation generated by Documentation Insight but I'd like to get that migrated to Wordpress too as Documentation Insight's editor is based on Internet Explorer.
![billede](https://github.com/graphics32/graphics32/assets/18141921/50876ffb-1a1d-4ff9-86ac-4bcf38118226)


Btw, have you seen that Graphics32 now supports SVG? 4450b7a :-)
![billede](https://github.com/graphics32/graphics32/assets/18141921/7350724b-1f9b-4132-bde9-113f870a40f8)

**Comment by AngusJohnson** [comment]
> I primarily wanted to get the source updated so your work wasn't lost. I will upload your updated documentation too but I'm not going to maintain it using DocProcessor going forward as I simply refuse to use notepad (or whatever) to main the documentation. Life's too short for that. I need WYSIWYG.

Yeah, the original DocProcessor I believe was written by Alex Denisov and then modified by several other contributors before I started meddling with it. My initial contribution was just to automate importing/converting file structures from new Graphics32 PAS units into HTML files, and only later started tweaking other things. My much more recent, and much more substantial changes were prompted by using DocProcessor to document my own software libraries. And that was because I was at least familiar with DocProcessor and its querks. 

> Btw, have you seen that Graphics32 now supports SVG?

No, I hadn't seen! I'm sure many Graphics32 users will find that really useful. My SVG code kind of grew out of proof of concept stuff, and I just kept adding extra features and workarounds for various bugs. So it really isn't well planned and I'd have completely rewritten it by now except that it will be a very big job. (It's still something I still plan to do, perhaps once I've finished another very big project I'm currently working on.)
